eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -85,13 +85,19 @@ stages:
                         condition: and(succeeded(), ne(variables['Skip.AutoAddTag'], 'true'))
                         displayName: 'Set Tag and Additional Tag'
 
+                      - script: |
+                          npm install $(Package.Archive)
```

Comment:
Do we expect this to always succeed? What if there is another package in the same pipeline that was just released but not fully indexed yet? 

In general we have been trying to depend on the smoke-test pipeline to catch packaging issues.

Reply:
After chatting we I think as long as we can successfully retry this step if it fails then it is probably adding enough value. I will say that I know a lot of developers hit the approve button on all the packages at the same time so that workflow will need to potentially be tweaked so we should at least let folks know they need to release them in order.